src/components/TreeImageScrubber.js

```diff
@@ -206,33 +207,25 @@ const TreeImageScrubber = (props) => {
   useEffect(() => {
     log.debug('mounted:');
     // update filter right away to prevent non-Filter type objects loading
+    document.title = `Verify - ${documentTitle}`;
     props.verifyDispatch.updateFilter(props.verifyState.filter);
     props.verifyDispatch.loadTreeImages();
-  }, [props.verifyDispatch]);
-
-  /* to update html document title */
-  useEffect(() => {
-    document.title = `Verify - ${documentTitle}`;
+    props.verifyDispatch.getTreeCount();
+    props.organizationDispatch.loadOrganizations();
```

Comment:
Triggering this loading here rather than in the components that use it means that `organizationList` is not necessarily populated when the component loads. In particular, you would need to open the _Verify_ page before you navigate to _Edit Planter_.

Reply:
True.  I see the reasoning for having each component load the state they need, especially if they are loaded from different places.
For the verify page there are many requests all happening when that page loads,  were two components making the same request and they are both updating a central state in the verify store.  I believe there are only two paths to load Edit Planter, through the verify page and through the planters page -- is there another route?

Perhaps, especially with the organization list, it would make sense to load this data when the user logs in, then the sub-components would make direct requests to load only if that state changed?

Reply:
True. I see the reasoning for having each component load the state they need, especially if they are loaded from different places.  

For the verify page, there are many requests all happening when that page loads, all updating the central stores.  Filter Top and the Edit Planter were duplicating the request for organization list and looked like they might be causing each other to re-render when that value updated in the store. (There are still random re-renders, possibly related to the species requests instead of the org list)

I can step back to components making the requests and see if I can manage the triggers to render based on checking the store for the values first.

With the organization list, it might make sense to load this data when the user logs in, set the store, then any sub-components relying on the data would make direct requests to load only if that state changed?

What do you think?

Reply:
I agree,loading once on login would make sense.

Reply:
Moved loading organizations to Account page which is first to load after login.  Is there a better place to put it?

Reply:
I don't think it makes sense to put the request in another component that doesn't use organizations.
I would suggest leaving it in the components that use it, but if you want to reduce duplication of API requests, having the model check whether it needs to make the request.